adapters/huaweiads/huaweiads.go

```diff
@@ -788,39 +778,54 @@ func convertCountryCode(country string) (out string) {
 	return defaultCountryName
 }
 
+func getCountryCodeFromMCC(MCC string) (out string) {
+	var countryMCC = strings.Split(MCC, "-")[0]
+	intVar, err := strconv.Atoi(countryMCC)
+
+	if err != nil {
+		return defaultCountryName
+	} else {
+		if result, found := MccList[intVar]; found {
+			return strings.ToUpper(result)
+		} else {
+			return defaultCountryName
+		}
+	}
+}
+
 // getDeviceID include oaid gaid imei. In prebid mobile, use TargetingParams.addUserData("imei", "imei-test");
 // When ifa: gaid exists, other device id can be passed by TargetingParams.addUserData("oaid", "oaid-test");
 // When getGaidFromDeviceIFA success, getDeviceIDFromUserExt failed, no error will be reported.
```

Comment:
Function `getGaidFromDeviceIFA` doesn't exist anymore, please modify comment accordingly

Reply:
Because "getGaidFromDeviceIFA" function is removed, the comment is removed also.